### PR TITLE
Feature/animemusic

### PR DESCRIPTION
**Pull request by AndrewStoddard**

### CODE REVIEW

File: src/database/anime.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            'CREATE TABLE IF NOT EXISTS anilistmedia (anilistMediaId INTEGER PRIMARY KEY, annId INTEGER, name TEXT)'
+        ).run();
+        db.prepare(
+            'CREATE TABLE IF NOT EXISTS anisong (anisongId INTEGER PRIMARY KEY, annId INTEGER, anilistMediaId INTEGER, url TEXT, songType TEXT, anisongType TEXT, animeEng TEXT, animeJap TEXT, songName TEXT, animeType TEXT)'
+        ).run();
+        const data: any = db.prepare('SELECT * FROM ann LIMIT 1').get();
+        if (!data) {
+            this.pullAllFromANN();
+        }
+        const job1 = new CronJob('0 0 0 * * *', this.pullRecentFromANN, null, true, 'America/New_York');
```
Comment (TRMoosee) [inline src/database/anime.ts:38]: Should day not be 0 here?

**Review by TRMoosee — APPROVED** [review]
Looks good, but please verify that the cron format is correct. I believe it should only be 5 values and not 6.

File: src/database/anime.ts (hunk 1 above)
Comment (AndrewStoddard) [reply src/database/anime.ts:38]: node-cron uses (second minute hour dayofmonth month dayofweek).
0 0 0 * * * should be daily at 12am EST
0 0 0 * * 1 should be Weekly, on Monday , at 12am EST

Ofc this is the first i have used node-cron, so dont take my word for it

**Review by TRMoosee — APPROVED** [review]
Looks good, looks like the cron npm module does use a 6 argument format.

**Pull request merged**